osmomath/math.go

```diff
@@ -8,6 +8,8 @@ import (
 // TODO: Analyze choice here.
 var powPrecision, _ = NewDecFromStr("0.00000001")
 
+const powIterationLimit = 150_000
```

Comment:
Curious, how was this chosen? 👀

Reply:
https://github.com/osmosis-labs/osmosis/pull/6627#discussion_r1345752782